ui/pages/Project/components/ProjectCollaborators.jsx

```diff
@@ -105,10 +105,11 @@ const CollaboratorContainer = styled.div`
   white-space: nowrap;
 `
 
-const ProjectCollaborators = React.memo(({ project, onSubmit }) => (
-  orderBy(project.collaborators, [c => c.hasEditPermissions, c => c.email], ['desc', 'asc']).map(c =>
+const ProjectCollaborators = React.memo(({ project, anvilCollaborator, onSubmit }) => (
+  <div>{anvilCollaborator && project.collaborators.filter(col => col.isAnvil === anvilCollaborator).length > 0 && <p>AnVIL Workspace Users</p>}{
```

Comment:
I'm surprised that the javascript linter isn't complaining about the indentations here. Can you double check the window where you are running npm to make sure its not showing any errors? Linting errors don't stop the code from working and don't cause tests to fail, so they can be easy to miss